xds/internal/balancer/cdsbalancer/cdsbalancer.go

```diff
@@ -669,3 +680,19 @@ func (ccw *ccWrapper) UpdateAddresses(sc balancer.SubConn, addrs []resolver.Addr
 	}
 	ccw.ClientConn.UpdateAddresses(sc, newAddrs)
 }
+
+// systemRootCertsProvider implements a certprovider.Provider that returns the
+// system default root certificates for validation.
+type systemRootCertsProvider struct{}
+
+func (systemRootCertsProvider) Close() {}
+
+func (systemRootCertsProvider) KeyMaterial(_ context.Context) (*certprovider.KeyMaterial, error) {
```

Comment:
Nit: since the context is the only parameter, can't we omit the `_` parameter name?

Reply:
Done.